external/db_drivers/liblmdb/mdb.c

```diff
@@ -176,6 +176,7 @@ typedef SSIZE_T	ssize_t;
 #include <semaphore.h>
 #elif defined(MDB_USE_SYSV_SEM)
 #include <sys/ipc.h>
+#define _WANT_SEMUN             1
```

Comment:
This change seems unrelated ... ?

Reply:
Sorry, I had no idea it would include subsequent commits in the PR. I've `git reset --force`'d away the last commit.